Cargo.toml

```diff
@@ -1,5 +1,2 @@
 [workspace]
 members = ["cnd", "libp2p-comit"]
-
-[replace]
```

Comment:
Please don't do two things in the same PR :clown_face:

Reply:
I'm still not sold on this @thomaseizinger, since we talked about how I do small refactorings/cleanup as the initial patches on a series (that becomes a single PR) I have tried _not_ doing this and making separate PRs.  It is a PITA because if the refactor/cleanup turns out to be wrong (when typically I would just rebase and ammend it) one has to go over to the other PR, fix it there, explain why it was wrong in the first place etc etc etc.  Also, any time while reviewing a PR, viewed on Github, if there is anything non-trivial in it I check out the branch and read each patch separately.  I've been doing this more since I noticed you were writing commit messages also :)  So I feel that this change is fine here since it is done as a separate patch at the start of the series.

Reply:
> Please don't do two things in the same PR 🤡

Is that a new rule? It's a separate commit.

Reply:
I was joking in reference to you guys picking apart my comit-crate refactoring :D

Back to being serious:

@tcharding 

It depends on the size, you can't come up with a hard rule ...
In this case I was joking (hence the clown), a little change like this should be fine (as long as it is in a separate commit).
Generally, if your refactorings have a bigger impact, you will have an easier life (less rebasing and conflicts) if you send them as separate PRs because they get merged faster.

Whether or not you actually need them for a change will depend on your forecasting abilities :)
If you don't trust them, don't send separate PRs until you are done (so you know).
You can also send them in one PR obviously. As mentioned above, I was only suggesting that as a solution to your complaints about having to constantly rebase your refactorings.
Try and optimize the following at the same time:

- make reviews as pleasant as possible
- make your own life as easy as possible